client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/design-picker-design-title.tsx

```diff
@@ -30,10 +44,20 @@ const DesignPickerDesignTitle: FC< Props > = ( { designTitle, selectedDesign } )
 		)
 	);
 
-	const bundleSettings = useBundleSettingsByTheme( selectedDesign.slug );
-
 	let badge: React.ReactNode = null;
-	if ( selectedDesign.software_sets && selectedDesign.software_sets.length > 0 ) {
```

Comment:
I think we need to keep the `software_sets` check at the beginning. Otherwise the Partner themes will show a badge with a weird layout:

Before | After
--- | ---
<img width="326" alt="Screenshot 2024-01-18 at 15 24 01" src="https://github.com/Automattic/wp-calypso/assets/1233880/97106bbd-b0ea-471f-94e7-ac633753657c"> | <img width="330" alt="Screenshot 2024-01-18 at 15 24 06" src="https://github.com/Automattic/wp-calypso/assets/1233880/229ed25b-f715-4844-8c7d-24e4d7f9b627">

Reply:
I've added spacing on https://github.com/Automattic/wp-calypso/pull/86582/commits/9a5973a6b81e2541cbc7f20688b4c3737d65383e. I think that the badge shown on this page should be the same(s) than on the listing.